_posts/2022-12-31-end-of-year-update.markdown

```diff
@@ -49,26 +49,26 @@ When Qt6 came around, they switched to CMake, which is a lot nicer for cross-com
 ### GTK+ 3 ([PR](https://github.com/managarm/bootstrap-managarm/pull/169))
 _PRed and written by [@Dennisbonke](https://github.com/Dennisbonke)_
 
-As mentioned in the [August 2022](https://managarm.org/2022/08/14/aug-update.html) update, we had GTK+ 2 working already. However, one major browser that was attempted required GTK+ 3, namely WebKitGTK. So, off we went on a challenge to get the required dependencies in. The big hassles proved to be GTK+ 3 itself, mainly due to runtime files missing, like GDK cache files, IM loader cache files, and the MIME database, which was fixed by adding post-install scripts and letting xbps configure all packages during boot, and DBus, which we still don't have fully working. The solution to DBus was quite simple, after we found out that DBus is only used for accessibility. We patched out the initialization calls for accessibility and after some locking fixes in mlibc we were able to run the `gtk3-demo` program successfully.
+As mentioned in the [August 2022](https://managarm.org/2022/08/14/aug-update.html) update, we had GTK+ 2 working already. However, one major browser that was attempted required GTK+ 3, namely WebKitGTK. So, off we went on a challenge to get the required dependencies in. The big hassles proved to be GTK+ 3 itself, mainly due to runtime files missing, like GDK cache files, IM loader cache files, and the MIME database, which was fixed by adding post-install scripts and letting xbps configure all packages during boot, and DBus, which we still do not have fully working. The solution to DBus was quite simple, after we found out that DBus is only used for accessibility. We patched out the initialization calls for accessibility and after some locking fixes in mlibc we were able to run the `gtk3-demo` program successfully.
 
 ![gtk3-demo](/assets/2022-end-of-year-update/gtk3-demo.png)
 
 ## Other projects updates
 
 ### mlibc on Linux ([repo](https://github.com/no92/linux-mlibc))
-While working towards running a proper browser on Managarm, we came to the realization that we can't rule out whether the kernel / servers are wrong (A.K.A. Managarm itself), or if mlibc had a bug. To eliminate the first possibility and granting easy access to lots of nice debug tools, we started the [mlibc LFS](https://github.com/no92/linux-mlibc) project, named after Linux From Scratch. It's essentially Linux From Scratch, but using mlibc instead of glibc.
+While working towards running a proper browser on Managarm, we came to the realization that we can not rule out whether the kernel / servers are wrong (A.K.A. Managarm itself), or if mlibc had a bug. To eliminate the first possibility and granting easy access to lots of nice debug tools, we started the [mlibc LFS](https://github.com/no92/linux-mlibc) project, named after Linux From Scratch. It is essentially Linux From Scratch, but using mlibc instead of glibc.
```

Comment:
'can not' is a bit weird

I would reword the first sentence: 'we struggled to determine whether mlibc was at fault, or the kernel / servers' (something like that)

Reply:
Also 'but using' -> 'but uses'

Reply:
Applied, thanks!